### PR TITLE
ENG-2058 new bundle installation with install plans

### PR DESCRIPTION
**Pull request by firegloves**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **firegloves**

If they are not already assigned, you can assign the PR to them by writing `/assign @firegloves` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-component-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-component-manager/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr111**

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr111**

**Comment by entando-jx** [comment]
@firegloves: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 141b70c103756e57f17a632fbc0426ae0b70ce9a | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@firegloves: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 4ff108b963299c22cc2a2f59df06bac874e88842 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@firegloves: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | d79168df754e070ea76bfed570def3dcc625294c | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr111**

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr111**

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr111**

File: src/main/java/org/entando/kubernetes/service/digitalexchange/job/EntandoBundleInstallService.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/kubernetes/service/digitalexchange/job/EntandoBundleInstallService.java:194]: @firegloves 

```
log.error("Error converting the received install plan to string", e);
```

**Review by w-caffiero-entando — CHANGES_REQUESTED** [review]

File: src/main/java/org/entando/kubernetes/client/model/assembler/InstallPlanAssembler.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/kubernetes/client/model/assembler/InstallPlanAssembler.java:1]: @firegloves revert this pls

avoid committing a file if there is not functional change

**Review by w-caffiero-entando — CHANGES_REQUESTED** [review]

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr111**

**Review by w-caffiero-entando — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-component-manager/pull/111#pullrequestreview-623126937" title="Approved">w-caffiero-entando</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-component-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando-k8s/entando-component-manager/blob/master/OWNERS)~~ [w-caffiero-entando]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
